src/iOS.Autofill/LockPasswordViewController.cs

```diff
@@ -33,22 +33,29 @@ public LockPasswordViewController(IntPtr handle)
 
         public override async void ViewDidLoad()
         {
-            _cancelButton = new UIBarButtonItem(UIBarButtonSystemItem.Cancel, CancelButton_TouchUpInside);
+            try
+            {
+                _cancelButton = new UIBarButtonItem(UIBarButtonSystemItem.Cancel, CancelButton_TouchUpInside);
 
-            base.ViewDidLoad();
+                base.ViewDidLoad();
 
-            _accountSwitchingOverlayHelper = new AccountSwitchingOverlayHelper();
+                _accountSwitchingOverlayHelper = new AccountSwitchingOverlayHelper();
 
-            _accountSwitchButton = await _accountSwitchingOverlayHelper.CreateAccountSwitchToolbarButtonItemCustomViewAsync();
-            _accountSwitchButton.TouchUpInside += AccountSwitchedButton_TouchUpInside;
+                _accountSwitchButton = await _accountSwitchingOverlayHelper.CreateAccountSwitchToolbarButtonItemCustomViewAsync();
+                _accountSwitchButton.TouchUpInside += AccountSwitchedButton_TouchUpInside;
 
-            NavItem.SetLeftBarButtonItems(new UIBarButtonItem[]
-            {
-                _cancelButton,
-                new UIBarButtonItem(_accountSwitchButton)
-            }, false);
+                NavItem.SetLeftBarButtonItems(new UIBarButtonItem[]
+                {
+                    _cancelButton,
+                    new UIBarButtonItem(_accountSwitchButton)
+                }, false);
 
-            _accountSwitchingOverlayView = _accountSwitchingOverlayHelper.CreateAccountSwitchingOverlayView(OverlayView);
+                _accountSwitchingOverlayView = _accountSwitchingOverlayHelper.CreateAccountSwitchingOverlayView(OverlayView);
+            }
+            catch (Exception ex)
+            {
+                LoggerHelper.LogEvenIfCantBeResolved(ex);
+            }
```

Comment:
If there is a crash here the user won't have an account switcher in the UI is that intended?

Reply:
Not intended, but I guess that's better than the Autofill extension crashing